.github/workflows/test.yml

```diff
@@ -88,8 +88,15 @@ jobs:
         sudo apt-get -y install qemu-user
     - name: build test wrapper
       run: ./tool/go build -o /tmp/testwrapper ./cmd/testwrapper
+    - name: build test wrapper
+      run: ./tool/go build -o /tmp/testwrapper ./cmd/testwrapper
     - name: test all
-      run: ./tool/go test ${{matrix.buildflags}} -exec=/tmp/testwrapper -bench=. -benchtime=1x ./...
+      run: ./tool/go test ${{matrix.buildflags}} -exec=/tmp/testwrapper
+      env:
+        GOARCH: ${{ matrix.goarch }}
+        GODEBUG: gocachetest=1
```

Comment:
leftover debug?

Reply:
yeah, I was just investigating from the question above, but now did that locally. It seems exec does affect test caching, so that'll be another step to look into.